### DISCUSSION TITLE
Change font/color (theme support)

### DISCUSSION THREAD

**Discussion by yermulnik** [ankitpokhrel/jira-cli#290] [Ideas] [answered]
Is it possible to change terminal font and color used by `jira-cli` or maybe there's already support to change themes? I didn't find either.

**Comment by ankitpokhrel** [comment, marked as answer]
Hi @yermulnik, font is not assigned by the tool so it should use whatever font you have configured in your terminal. Colors are hardcoded and not configurable at the moment.

**Comment by yermulnik** [comment]
Got you. Thanks.
Terminal font and colors that I use unfortunately do not show all the chars `jira-cli` uses internally and line with selector isn't visible. Was hoping it can be adjusted via `jira-cli` config.
Thanks for the great tool anyways 👍🏻
